### PR TITLE
Add Test and UITest check

### PR DESCRIPTION
**Pull request by BirmacherAkos**
Depends on:
https://github.com/bitrise-io/xcode-project/pull/34

---

Add isTestProduct() & isUITestProduct for target.go

### CODE REVIEW

File: xcodeproj/target.go [hunk 1]
```diff
@@ -73,7 +73,15 @@ func (t Target) IsAppExtensionProduct() bool {
 
 // IsExecutableProduct ...
 func (t Target) IsExecutableProduct() bool {
```
Comment (godrei) [inline xcodeproj/target.go]: with this change we will add ui test targets to the xcode-archive step generated exportOptions.plist:
- https://github.com/bitrise-steplib/steps-xcode-archive/blob/master/utils/entitlements.go#L78
- https://github.com/bitrise-io/xcode-project/blob/5929ad85d89233b3296d2d795e068a656db7fb25/xcodeproj/target.go#L51

i think i would case an issue

Comment (BirmacherAkos) [reply xcodeproj/target.go]: Fixed

File: xcodeproj/target.go [hunk 2]
```diff
@@ -76,6 +76,14 @@ func (t Target) IsExecutableProduct() bool {
 	return t.IsAppProduct() || t.IsAppExtensionProduct()
 }
 
+func (t Target) isTestProduct() bool {
```
Comment (godrei) [inline xcodeproj/target.go]: Should not we expose this function for client-side?

Comment (BirmacherAkos) [reply xcodeproj/target.go]: Oh

Comment (BirmacherAkos) [reply xcodeproj/target.go]: Fixed

**Pull request merged**